docs/src/services/relay.md

```diff
@@ -1,9 +1,9 @@
 # Control Plane Relay
 
-| services | ports | Protocol |
-|----------|-------|-----------|
-| ADS | 7800 | gRPC(IPv4) |
-| CPDS | 7900 | gRPC(IPv4) |
+| services | ports | Protocol           |
+|----------|-------|--------------------|
+| ADS      | 7800  | gRPC(IPv4 OR IPv6) |
+| CPDS     | 7900  | gRPC(IPv4 OR IPv6) |
```

Comment:
One thing I noted when doing the examples is that it's not just `quilkin relay` it's `quilkin relay (agones|file)` (at least from the command prompt).

And in the example for Agones - we do run `quilkin relay agones`. I am wondering if that's because we still want to pickup the Filter ConfigMap at the Relay level, but the endpoint configuration at the Agent level?

But it does seem like we have the same CLI args shared between `quilkin relay` and `quilkin agent`, which may be confusing (Also, noting that `quilkin relay` with no extra args works will run, but I assume will skip any of the provider/agent specific logic.

For example, `quilkin relay agones --config-namespace="gameservers"` is valid, but I'm guessing that `--config-namespace` doesn't actually do anything? (maybe to get removed later?). Same for `quilkin relay file` ?

Just wanted to make sure I wasn't missing anything here when writing docs. I figure this is just an artifact of work in progress, but wanted to check.

Reply:
Yeah that's an interesting point. I don't hugely worry about the cli being dot seperated, but also don't hate it 😄 

But I'll also add some notes to the agent/quickstart around relay sub-command functionality so that is also clear.

Just so I make sure to get that right:

`quilkin relay --gameservers-namespace` .... that doesn't do anything does it? Since the relay isn't looking at Agones `GameServer` data locally? (or does it???)

I also assume `quilkin relay file <path>`  wouldn't actually do anything, since the full config would come from seperate `quilkin agents` - or would it grab the local file info and merge it with whatever else is coming in?